### PR TITLE
Small cleanup.

### PR DESCRIPTION
**Pull request by snakehand**
Fixed some compiler warnings (dyn Error)
Made code somewhat more idiomatic
Cleaned up some deep nesting

### CODE REVIEW

File: src/repo.rs [hunk 1]
```diff
@@ -3,35 +3,28 @@ use std::process::{Command, Stdio};
...
-            let git_path = dir.join(".git/config");
-
-            match Ini::load_from_file(git_path) {
-                Ok(file) => {
-                    for (sec, prop) in file.iter() {
-                        if let Some(sec) = sec {
-                            if sec != "remote \"origin\"" {
-                                continue;
-                            }
+    let dir = match std::env::current_dir() {
```
Comment (kollstrom) [inline src/repo.rs:6]: Nitpicking here, but is it somewhat more "idiomatic" to add `use std::env`  and shorten this line to `env::current_dir()`?

File: src/shellout.rs [hunk 2]
```diff
@@ -24,7 +24,9 @@ fn exec_lpass_input(args: &[&str], data: &str) -> Result<String, LPassError> {
     }
     let mut child = child.unwrap();
 
-    child.stdin.as_mut().unwrap().write_all(data.as_bytes());
+    if let Err(err) = child.stdin.as_mut().unwrap().write_all(data.as_bytes()) {
+        return Err(LPassError::new("Could write data", Some(Box::from(err))));
```
Comment (kollstrom) [inline src/shellout.rs]: Should the string be _Could not write data_ here?

**Review by kollstrom — CHANGES_REQUESTED** [review]
Thanks for your contribution, @snakehand and for improving the `hamr` codebase!
I've suggested some tiny changes for us to discuss

File: src/shellout.rs (hunk 2 above)
Comment (snakehand) [reply src/shellout.rs]: You are correct - I guess I was a little quick to correct here when I got compiler warning.

**Pull request merged**